### ISSUE TITLE
Way to change TextBox.Text binding default UpdateSourceTrigger to PropertyChanged?

### ISSUE THREAD

**Issue by omnilogix** [microsoft/microsoft-ui-xaml#5414]
In WPF we could do `TextBox.TextProperty.OverrideMetaData... `to set the default binding behavior.  Is there such a method available in WinUI?

I would like the default UpdateSourceTrigger for TextBox bindings to the the Text property to be PropertyChanged, instead of LostFocus.

**Comment by StephenLPeters** [comment]
No, I don't think this exists for xaml.  @MikeHillberg ?

**Comment by MikeHillberg** [comment]
Right, WinUI doesn't have an equivalent to [FrameworkPropertyMetadata.DefaultUpdateSourceTrigger](https://docs.microsoft.com/dotnet/api/System.Windows.FrameworkPropertyMetadata.DefaultUpdateSourceTrigger).

**Comment by omnilogix** [comment]
@MikeHillberg  Feature request?  I literally have thousands of WPF TextBoxes to port.  This is very useful in LOB apps where all bindings need to be up to date prior to save button (or accelerator) which doesn’t trigger binding updates from focus change, which can leave un-committed data during save.  It’s easy to miss the inclusion of the UpdateSourceTrigger with so many in one app.  A way to set default  behavior would be a really big benefit here.  Something that could be set via Style at app level perhaps

**Comment by tuggernuts** [comment]
Any chance of thawing out this request? I know Winui has lots of issues, but LOB apps are still being developed in 2022. @omnilogix 's comment is dead on.
